generators/environment/templates/bin/pre-install.sh

```diff
@@ -10,11 +10,6 @@
 # of `grunt install`.
 ##
 
-if [ ! -f src/sites/default/default.settings.php ]; then
-  echo "Please run pre-install after successfully building the docroot."
-  exit 1
-fi
```

Comment:
Yes, or if they ran rm -Rf build/html to do a hard clean. That line is there specifically to make the process end ASAP if there's no point in continuing. I realize the conditionals are slightly similar, but in this case I deliberately included both because they are not identical, and it helps clarify there are cases to skip executing that are different than the case to skip the copy.